itests/org.openhab.binding.max.tests/.project

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>org.openhab.binding.wemo.test</name>
-	<comment>This is the test component for Eclipse SmartHome Wemo Binding</comment>
+	<name>org.openhab.binding.max.tests</name>
```

Comment:
wemo / max?

Reply:
I'll remove it. Eclipse likes to annoy me by readding project files when the projects are still open but no longer exist on the branch. :smile: